libp2p/protocols/kademlia/keys.nim

```diff
@@ -36,7 +48,7 @@ proc getBytes*(k: Key): seq[byte] =
     case k.kind
     of KeyType.PeerId:
       k.peerId.getBytes()
-    of KeyType.Raw, KeyType.Unhashed:
+    of KeyType.Raw, KeyType.Unhashed, KeyType.Hashed:
```

Comment:
possibly not neccisary. if it is, probably best for another pr, as what a hashed key looks like should probably be defined by the application, making this feature non-trivial enough to be its own efforts